apptools/persistence/state_pickler.py

```diff
@@ -130,7 +129,7 @@ def gunzip_string(data):
     """Given a gzipped string (`data`) this unzips the string and
     returns it.
     """
-    if PY_VER== 2 or (bytes is not str and type(data) is bytes):
+    if (bytes is not str and type(data) is bytes):
```

Comment:
Umm. `bytes is not str` should always return `True` in Python 3. We should be able to get rid of it.